src/parsec/util.hpp

```diff
@@ -71,6 +73,20 @@ namespace cbdc::parsec {
                  broker::key_type key,
                  broker::value_type value,
                  const std::function<void(bool)>& result_callback) -> bool;
+
+    /// Asynchronously get the value stored at key from the cluster.
+    /// Intended for testing purposes.
```

Comment:
Nit-picky but potentially administrative functions as well like migrating keys? I wouldn't use that example but maybe the comment should say "testing and administration purposes" instead?

Yes, I did just add a review comment on a code comment... :)

Reply:
I can get on board with that, will update

Reply:
For context, this comment is trying to convey that this is not the optimal method for reading from the shards as a system component, it is better to handle shard responses in the agent